spec/services/ial_context_spec.rb

```diff
@@ -129,6 +153,52 @@
     end
   end
 
+  describe '#bill_for_ial_1_or_2' do
+    context 'when ial is nil' do
+      let(:ial) { nil }
+      it { expect(ial_context.bill_for_ial_1_or_2).to eq(1) }
+    end
+
+    context 'when ial1' do
+      let(:ial) { Idp::Constants::IAL1 }
+      it { expect(ial_context.bill_for_ial_1_or_2).to eq(1) }
+    end
+
+    context 'when ial1' do
+      let(:ial) { Idp::Constants::IAL2 }
```

Comment:
this comment is a lie
```suggestion
    context 'when ial2' do
      let(:ial) { Idp::Constants::IAL2 }
```